jsapp/scss/main.scss

```diff
@@ -20,6 +20,7 @@
 @import "./components/kobo.text-button";
 @import "./components/kobo.checkbox-and-radio";
 @import "./components/kobo.toggle-switch";
+@import "./components/kobo.multi-checkbox";
 @import "./components/kobo.modal";
 @import "./components/kobo.tooltips";
 @import "./components/kobo.navigation";
```

Comment:
Notes to self from internal discussion:

- Disallow self attachment
- De-load the modal